### PR TITLE
Release 1.8.0

### PR DESCRIPTION
**Pull request by tnix100**

### CODE REVIEW

**Comment by showierdata9978** [comment]
nevermind did not see it was a different part

**Review by MikeDev101 — APPROVED** [review]

File: rest_api/__init__.py [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+from .posts import posts_bp
+from .users import users_bp
+from .chats import chats_bp
+from .search import search_bp
+from .admin import admin_bp
+
+
+# Init app
+app = Quart(__name__, static_folder="static")
+app.config["APPLICATION_ROOT"] = "/api"
```
Comment (showierdata9978) [inline rest_api/__init__.py:16]: looks like you forgot to change it from /api/

**Review by showierdata9978 — APPROVED** [review]
other then that lgtm

**Comment by theotherhades** [comment]
Meower Server: Adventure Update (you're welcome for my incredibly constructive comment)

**Review by showierdata9978 — APPROVED** [review]
LGTM

**Review by MikeDev101 — APPROVED** [review]

**Pull request merged**